src/Accounts/Authentication.ResourceManager/Models/PSAzureEnvironment.cs

```diff
@@ -217,6 +219,16 @@ public bool OnPremise
         /// </summary>
         public string GraphEndpointResourceId { get; set; }
 
+        /// <summary>
+        /// Gets or sets the Uri of the Microsoft Graph endpoint.
```

Comment:
`MicrosoftGraphUrl` amd `MicrosoftGraphEndpointResourceId` are not defined in `IAzureEnvironment `